.env.example

```diff
@@ -6,3 +6,6 @@ SPOTIFY_CLIENT_SECRET=
 
 # Optional
 # CACHE_LIMIT=2GB
+# BOT_STATUS=
```

Comment:
Could you add a comment with a link to the possible values for these variables?

A section should also be added to the README about them.

Reply:
Both done :)